pkp-sign-openai/nodejs/src/litAction.ts

```diff
@@ -0,0 +1,48 @@
+// Filename: litAction.ts
+// @ts-nocheck
+const _litActionCode = async () => {
+  try {
+    const toSign = await LitActions.runOnce(
+      { waitForResponse: true, name: "Lit Actions Test" },
+      async () => {
+        const messages = [
+          {
+            role: "system",
+            content:
+              "You are an AI assistant. Only answer with a single sentence.",
+          },
+          { role: "user", content: "Is Lit Protocol good?" },
+        ];
+        const response = await fetch(
+          "https://api.openai.com/v1/chat/completions",
+          {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+              Authorization: `Bearer ${apiKey}`,
+            },
+            body: JSON.stringify({ model: "gpt-4", messages }),
```

Comment:
```suggestion
            body: JSON.stringify({ model: "gpt-4o-mini", messages }),
```

The README says `gpt-4o-mini` is being used